go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/check-payload
 
-go 1.19
+go 1.23.0
```

Comment:
should be 1.23

Reply:
Huh. I thought 1.23 had to have a z-stream specified. 😕 I'll update it.

Reply:
In fact it should be 1.23.0 (they have changed the notation at around go 1.20 or 1.21).

Reply:
I think 1.23 is correct based on the [docs](https://go.dev/doc/modules/gomod-ref#go).

Reply:
It's weird--I could have sworn a different upgrade to 1.23 was upset with me that there wasn't a z-stream, but I tried it here and it all seems to be fine? I'll give it a shot at least.

Reply:
So, `1.23` means *any* 1.23 release, including `rc` etc. `1.23.0` means the same thing but without rcs, betas etc. The difference is subtle and probably doesn't matter in practice, but I saw that recently go tools start to add the .z suffix.